_layouts/post.html

```diff
@@ -1,19 +1,7 @@
 ---
-layout: default
+layout: page
 main:
   class: usa-grid usa-section usa-content usa-layout-docs
 ---
-{% assign sidenav = site.data.navigation[page.sidenav] | default: page.sidenav %}
-{% if sidenav %}
-<aside class="usa-width-one-fourth usa-layout-docs-sidenav">
-  {% include sidenav.html links=sidenav %}
-</aside>
 
-<div class="usa-width-three-fourths usa-layout-docs-main_content">
-  {{ content }}
-</div>
-{% else %}
-<div class="usa-layout-docs-main_content">
-  {{ content }}
-</div>
-{% endif %}
```

Comment:
⭐️ bonus: updating the post layout to inherit from page. 
_(The code was identical between the post and page layouts)_